### ISSUE TITLE
[FR]Storage Manager: Add transfer acceleration option 

### ISSUE THREAD

**Issue by rinald-ta** [aws-amplify/amplify-ui#5163]
### On which framework/platform would you like to see this feature implemented?

React

### Which UI component is this feature-request for?

Storage (Storage Manager)

### Please describe your feature-request in detail.

Right now, there doesn't seem to be an option to use the transfer acceleration endpoint when uploading files via the `StorageManager` component.

Storage APIs have an optional `useAccelerateEndpoint`, e.g.

```ts
import { uploadData } from 'aws-amplify/storage';

uploadData({
  key,
  data: file,
  options:  {
    useAccelerateEndpoint?: boolean; // Whether to use accelerate endpoint.
  }
});
```

### Please describe a solution you'd like.

I would like to have the option of using transfer acceleration with the `StorageManager` component

```tsx
import { StorageManager } from '@aws-amplify/ui-react-storage'

const Component = () => {
  return (
    <StorageManager
      accessLevel="guest"
      maxFileCount={1}
      useAccelerateEndpoint={true}
    />
  )
}
```

### We love contributors! Is this something you'd be interested in working on?

- [ ] 👋 I may be able to implement this feature request.
- [ ] ⚠️ This feature might incur a breaking change.

**Comment by esauerbo** [comment]
@rinald-ta thank you for your feature request, we'll consider this in our roadmap planning. If others are interested please thumbs up this issue.

**Comment by rinald-ta** [comment]
@esauerbo Should we close this issue. From my understanding, this has been released already.

**Comment by esauerbo** [comment]
@rinald-ta yes, thanks for calling that out! This has been released in `@aws-amplify/ui-react-storage@3.2.0`, closing this out.
